### PR TITLE
Installation for Raspberry Pi

### LINKED ISSUE #873: Document installation for Raspberry Pi
I spent a lot of time googling installation instructions for the Raspberry Pi, many results were outdated and many recommended compiling Erlang from source (not ideal on a slow ARM processor).

Trying to install the Debian package from https://www.erlang-solutions.com/resources/download.html for Raspbian gave me the following error:

```
$ dpkg -i esl-erlang_19.1-1\~raspbian\~jessie_armhf.deb
dpkg: error processing archive esl-erlang_19.1-1~raspbian~jessie_armhf.deb (--install):
 package architecture (amd64) does not match system (armhf)
Errors were encountered while processing:
 esl-erlang_19.1-1~raspbian~jessie_armhf.deb
```

So that package seems to be incorrectly packaged for amd64.

In the end I found http://www.antonfagerberg.com/blog/elixir-on-raspberry-pi/ and learned that the Erlang Solutions repository contains packages for `armhf`.

Replacing `wheezy` with `jessie` (latest Raspbian is `jessie`) in the instructions from the blog post, I ended up doing the following:
```
echo "deb http://packages.erlang-solutions.com/debian jessie contrib" > /etc/apt/sources.list.d/erlang-solutions.list
wget http://packages.erlang-solutions.com/debian/erlang_solutions.asc
sudo apt-key add erlang_solutions.asc
apt update
apt install elixir
```

Given the popularity of Raspberry Pis, I think this should be better documented on http://elixir-lang.org/install.html.

### PR DESCRIPTION
**Pull request by railsnewbie257**
Came across Issue #873 and thought the suggestion for adding documentation for Raspberry Pi would be useful. I just used it myself.  Also, noticed the Issue has been dormant for a few months so thought to pick it up.

The erlang download is especially useful.

Closes #873

### CODE REVIEW

File: install.markdown [hunk 1]
```diff
@@ -58,6 +58,23 @@ If your distribution contains an old Elixir/Erlang version, see the sections bel
   * Chocolatey
     * `cinst elixir`
 
+### Raspberry Pi
+If necessary replace "jessie" with the name of your Raspian release
```
Comment (josevalim) [inline install.markdown]: Minor detail but can you please add a new line after the title? Thank you!

Comment (railsnewbie257) [inline install.markdown]: Extra line added and commits squashed.
Thanks for the review on a weekend.

File: install.markdown [hunk 2]
```diff
@@ -58,6 +58,24 @@ If your distribution contains an old Elixir/Erlang version, see the sections bel
   * Chocolatey
     * `cinst elixir`
 
+### Raspberry Pi
+
+If necessary replace "jessie" with the name of your Raspian release
```
Comment (whatyouhide) [inline install.markdown]: I think we want a trailing period on this sentence.

File: install.markdown [hunk 3]
```diff
@@ -58,6 +58,24 @@ If your distribution contains an old Elixir/Erlang version, see the sections bel
...
     * `cinst elixir`
 
+### Raspberry Pi
+
+If necessary replace "jessie" with the name of your Raspian release   
+  
+  * The Erlang Solutions repository has a prebuilt package for armhf.  
+    This saves significant time instead of recompiling natively.  
+  
+  * Get erlang key
```
Comment (whatyouhide) [inline install.markdown]: `erlang` -> `Erlang`

File: install.markdown (hunk 2 above)
Comment (railsnewbie257) [inline install.markdown]: Trailing "."(period) added

File: install.markdown (hunk 3 above)
Comment (railsnewbie257) [inline install.markdown]: erlang->Erlang (capitalized)

**Comment by railsnewbie257** [comment]
Second set of revisions done.
Commit squashed.

**Comment by whatyouhide** [comment]
Thanks @railsnewbie257!

**Pull request merged**